signingscript/docker.d/init_worker.sh

```diff
@@ -106,6 +106,12 @@ case $ENV in
         test_var_set 'AUTOGRAPH_OMNIJA_USERNAME'
         test_var_set 'AUTOGRAPH_WIDEVINE_PASSWORD'
         test_var_set 'AUTOGRAPH_WIDEVINE_USERNAME'
+        test_var_set 'AUTOGRAPH_FENIX_PASSWORD'
+        test_var_set 'AUTOGRAPH_FENIX_USERNAME'
+        test_var_set 'AUTOGRAPH_FENIX_MOZILLA_ONLINE_PASSWORD'
+        test_var_set 'AUTOGRAPH_FENIX_MOZILLA_ONLINE_USERNAME'
```

Comment:
These don't seem to be actually used, right?

Reply:
Right. I was just blindly copying. Updated now.